data/presets/man_made/gantry.json

```diff
@@ -0,0 +1,26 @@
+{
+    "icon": "temaki-toll_gantry",
+    "fields": [
+        "layer"
+    ],
+    "moreFields": [
+        "material",
+        "maxheight",
+        "operator",
+        "ref"
+    ],
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "man_made": "gantry"
+    },
+    "terms": [
+        "gantry",
+        "highway gantry",
+        "sign holder",
+        "road sign holder",
+        "sign structure"
+    ],
+    "name": "Road Sign Structure"
```

Comment:
Do we need a more generic name, such as “gantry”? The [original proposal](https://wiki.openstreetmap.org/wiki/Proposed_features/Gantry) suggests using this tag for railroad gantries as well as road sign gantries, and it may not be obvious that a “road sign structure” can hold traffic lights, an electronic toll collection device, or a traffic counter.

Reply:
I changed the name to `Gantry` and added presets for `power=portal` (gantry tower) and `crane:type=gantry_crane`